properties/P000138.md

```diff
@@ -0,0 +1,10 @@
+---
+uid: P000138
+slug: countable-continuous-maps
+name: Countably-many continuous self-maps
+refs:
+  - mo: 418619
+    name: Is there an infinite topological space with only countably many continuous functions to itself?
+---
+
+A space with only countably-many distinct maps from the space to itself.
```

Comment:
No need for the word "distinct".  But "continuous maps" would be good to avoid ambiguity for the reader.